Document truly transactional emails v2 feature (#723)
mautic/user-documentation: docs/contacts/frequency_rules.rst

How to set frequency rules
***************************

.. vale off

* Globally, from the configuration panel, you can set frequency rules for both Email and SMS settings.

* Individually, from a Contact's detail page under the dropdown menu on the upper right-hand side, you can select the Channels where you want the rules to apply. Setting the rule here overrides the general settings.
* 

> 

Reviewer: Remove this empty bullet point.

```suggestion

```

.. vale on

.. vale off

Emails and frequency rules
**************************

> .. vale on

Reviewer: Remove the vale off/on statements.

```suggestion
Emails and frequency rules
**************************
```

> Frequency rules determine whether an Email counts towards the Contact's communication limits. This behavior depends on the **Send to unsubscribed contacts** setting configured in the Email's Advanced tab.

Reviewer: ```suggestion
.. vale off

Frequency rules determine whether an Email counts towards the Contact's communication limits. This behavior depends on the **Send to unsubscribed contacts** setting configured in the Email's Advanced tab.
```

> * When **Send to unsubscribed contacts** is set to **No**, the Email counts towards the Contact's frequency rule limits. If a Contact has reached their limit, Mautic postpones the Email until the limit resets.

Reviewer: ```suggestion
When you set **Send to unsubscribed contacts** to **No**, the Email counts towards the Contact's frequency rule limits. If a Contact has reached their limit, Mautic postpones the Email until the limit resets.
```

> When you set **Send to unsubscribed contacts** to **No**, the Email counts towards the Contact's frequency rule limits. If a Contact has reached their limit, Mautic postpones the Email until the limit resets.

Reviewer: ```suggestion
* When you set **Send to unsubscribed contacts** to **No**, the Email counts towards the Contact's frequency rule limits. If a Contact has reached their limit, Mautic postpones the Email until the limit resets.

.. vale on
```

> * When you set **Send to unsubscribed contacts** to **Yes**, the Email doesn't count towards frequency rule limits. This ensures that Mautic delivers important transactional communications regardless of how many other Emails the Contact has received.

Reviewer: ```suggestion
* When you set **Send to unsubscribed contacts** to **Yes**, the Email doesn't count towards frequency rule limits. This ensures that Mautic delivers important transactional communications regardless of how many other Emails the Contact has received.

.. vale on
```

> * Globally, from the configuration panel, you can set frequency rules for both Email and SMS settings.

Reviewer: ```suggestion
.. vale off

* Globally, from the configuration panel, you can set frequency rules for both Email and SMS settings.
```

> * Individually, from a Contact's detail page under the dropdown menu on the upper right-hand side, you can select the Channels where you want the rules to apply. Setting the rule here overrides the general settings.

Reviewer: ```suggestion
* Individually, from a Contact's detail page under the dropdown menu on the upper right-hand side, you can select the Channels where you want the rules to apply. Setting the rule here overrides the general settings.
* 

.. vale on
```

.. vale off

Emails and frequency rules
**************************

.. vale on

Frequency rules determine whether an Email counts towards the Contact's communication limits. This behavior depends on the **Send to unsubscribed contacts** setting configured in the Email's Advanced tab.

> * When **Send to unsubscribed contacts** is set to **No**, the Email counts towards the Contact's frequency rule limits. If a Contact has reached their limit, Mautic postpones the Email until the limit resets.

Reviewer: Use active instead of passive voice for "is set".

> * When **Send to unsubscribed contacts** is set to **Yes**, the Email doesn't count towards frequency rule limits. This ensures that important transactional communications are delivered regardless of how many other Emails the Contact has received.

Reviewer: Use active instead of passive voice.

> * When you set **Send to unsubscribed contacts** to **Yes**, the Email doesn't count towards frequency rule limits. This ensures that Mautic delivers important transactional communications regardless of how many other Emails the Contact has received.

Reviewer: ```suggestion
.. vale off

Frequency rules determine whether an Email counts towards the Contact's communication limits. This behavior depends on the **Send to unsubscribed contacts** setting configured in the Email's Advanced tab.

* When **Send to unsubscribed contacts** is set to **No**, the Email counts towards the Contact's frequency rule limits. If a Contact has reached their limit, Mautic postpones the Email until the limit resets.

* When you set **Send to unsubscribed contacts** to **Yes**, the Email doesn't count towards frequency rule limits. This ensures that Mautic delivers important transactional communications regardless of how many other Emails the Contact has received.

.. vale on
```

> For more information about the **Send to unsubscribed contacts** setting, see :doc:`/channels/emails`.

Reviewer: We can link directly to the subsection for easy access.

```suggestion
See :ref:`send to unsubscribed Contacts` for more information.
```
